packages/app-project/src/shared/components/SubjectPreview/SubjectPreview.js

```diff
@@ -5,84 +5,87 @@ import { FavouritesButton, Media } from '@zooniverse/react-components'
 import { CollectionsButton, TalkLink } from './components'
 
 const defaultSubject = {
-  favorite: false,
-  id: '',
-  toggleFavorite: () => false,
-  locations: []
+	favorite: false,
+	id: '',
+	toggleFavorite: () => false,
+	locations: []
 }
-function SubjectPreview ({
-  height = '250px',
-  isLoggedIn = false,
-  placeholder,
-  slug,
-  subject = defaultSubject,
-  width = '400px'
+function SubjectPreview({
+	height = '250px',
+	isLoggedIn = false,
+	placeholder,
+	slug,
+	subject = defaultSubject,
+	width = '400px'
 }) {
-  const subjectURLs = subject.locations.map(location => Object.values(location)[0])
-  const subjectURL = subjectURLs[0]
-  const collectionsModal = createRef()
-  const href = `/projects/${slug}/talk/subjects/${subject.id}`
+	const subjectURLs = subject.locations.map(location => Object.values(location)[0])
+	const subjectURL = subjectURLs[0]
+	const collectionsModal = createRef()
+	const href = `/projects/${slug}/talk/subjects/${subject.id}`
 
-  return (
-    <Box
-      fill
-    >
-      <Anchor
-        href={href}
-      >
-        <Box
-          align='center'
-          height={height}
-          overflow='hidden'
-          width={width}
-        >
-          <Media
-            alt={`subject ${subject.id}`}
-            height={700}
-            placeholder={placeholder}
-            src={subjectURL}
-            width={700}
-          />
-        </Box>
-      </Anchor>
-      <Box direction='column' gap='xsmall' pad={{ vertical: 'xsmall' }}>
-        <TalkLink
-          href={href}
-        />
-        <FavouritesButton
-          checked={subject.favorite}
-          disabled={!isLoggedIn}
-          onClick={subject.toggleFavourite}
-        />
-        <CollectionsButton
-          disabled={!isLoggedIn}
-          subject={subject}
-        />
-      </Box>
-    </Box>
-  )
+	return (
+		<Box
+			fill
+		>
+			<Anchor
+				role="subject-preview-link"
+				href={href}
+			>
+				<Box
+					align='center'
+					height={height}
+					overflow='hidden'
+					width={width}
+					role="shit"
+				>
+					<Media
+						role="subject-preview-media"
```

Comment:
Another invalid role.

Reply:
Same as above.. How about using the Title attribute to specify _this particular_ media element. There's also a quirk with the tests where it won't find a Media element by it's alt tag. Here's a snippet from `screen.debug()` in the `SubjectPreview.spec.js` file.

`
<a
          class="StyledAnchor-sc-1rp7lwl-0 epna-Df"
          href="/projects/zooniverse/snapshot-serengeti/talk/subjects/123"
          title="subject 123 preview link"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 cnzmCK"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 RwhTp"
              controls=""
              title="subject 123 image"
            />
            <noscript />
          </div>
        </a>
`

Reply:
As per above comment regarding not using the title... How about using `data-testid` instead for this?